src/octree/mod.rs

```diff
@@ -133,94 +133,92 @@ pub struct Octree {
     nodes: FnvHashMap<NodeId, NodeMeta>,
 }
 
-pub struct PointsInBoxIterator<'a> {
-    octree: &'a Octree,
-    aabb: &'a Aabb3<f64>,
-    intersecting_nodes: Vec<NodeId>,
+fn get_node_iterator(octree: &Octree, node_id: &NodeId) -> NodeIterator {
```

Comment:
Would it be awkward to make this a method of `Octree`?

Reply:
You mean a private or public member?
I wouldn't make it public, because this is something nobody outside should need to use directly... if private, I don't have a tendency what's better...